docs/aci-compose-features.md

```diff
@@ -16,75 +16,81 @@ __Legend:__
-| service.pid                    | x |  
-| service.ports                  | ✓ |  Only symetrical por mapping is supported in ACI. See #exposing-ports.
-| service.secrets                | ✓ |  See #secrets.
-| service.security_opt           | x |  
-| service.stop_grace_period      | x |  
-| service.stop_signal            | x |  
-| service.sysctls                | x |  
-| service.ulimits                | x |  
-| service.userns_mode            | x |  
-| service.volumes                | ✓ |  Mapped to AZure File Shares. See #persistent-volumes.
+| service.pid                    | x |
+| service.ports                  | ✓ |  Only symetrical por mapping is supported in ACI. See [Exposing ports](#exposing-ports).
+| service.secrets                | ✓ |  See [Secrets](#secrets).
+| service.security_opt           | x |
+| service.stop_grace_period      | x |
+| service.stop_signal            | x |
+| service.sysctls                | x |
+| service.ulimits                | x |
+| service.userns_mode            | x |
+| service.volumes                | ✓ |  Mapped to AZure File Shares. See [Persistent volumes](#persistent-volumes).
 | service.restart                | x |  Replaced by service.deployment.restart_policy
-|                                |   |                                                              
-| __Volume__                     | x |                                                              
-| driver                         | ✓ |  See #persistent-volumes.
-| driver_opts                    | ✓ |  
-| external                       | x |  
-| labels                         | x |  
-|                                |   |  
-| __Secret__                     | x |  
-| TBD                            | x |  
-|                                |   |  
-| __Config__                     | x |  
-| TBD                            | x |  
-|                                |   |  
-              
+|                                |   |
+| __Volume__                     | x |
+| driver                         | ✓ |  See [Persistent volumes](#persistent-volumes).
+| driver_opts                    | ✓ |
+| external                       | x |
+| labels                         | x |
+|                                |   |
+| __Secret__                     | x |
+| TBD                            | x |
+|                                |   |
+| __Config__                     | x |
+| TBD                            | x |
+|                                |   |
+
+
+## Logs
+
+Container logs can be obtained for each container with `docker logs <CONTAINER>`.
+The Docker ACI integration does not currently support aggregated logs for containers in a compose application,
+due to ACI not providing either aggregated logs for a container group, or log timestamps that can be easily used to reorder logs properly for display.
```

Comment:
Not sure we need this line. We can create an issue, link it above and see if users thumbs up it